container-engine-lib/lib/backend_impls/kubernetes/kubernetes_manager/kubernetes_manager.go

```diff
@@ -1867,15 +1867,7 @@ func (manager *KubernetesManager) waitForPodAvailability(ctx context.Context, na
 				podStateStr,
 			)
 		case apiv1.PodSucceeded:
-			podStateStr := manager.getPodInfoBlockStr(ctx, namespaceName, pod)
```

Comment:
can we move this condition to the top of the list near PodRunning and PodUnknown for consistency.

Reply:
Sure